### PR TITLE
Radio Button Component Fixes

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

Fixing radio buttons:
would have to double click if something else was selected
would need to tabs to get to radio buttons
couldnt go between radio buttons with arrow keys

## Why?

NAC-1242

## This PR has:

- [x] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [x] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

## Anything else?
image

### CODE REVIEW

File: query-tool/src/Piipan.QueryTool/package.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
     "autoprefixer": "^10.4.8",
     "cypress": "^10.4.0",
     "cypress-audit": "^1.1.0",
+    "cypress-plugin-tab": "^1.0.5",
```
Comment (dborgen2) [inline query-tool/src/Piipan.QueryTool/package.json:17]: You can probably drop this now and just use the realPress("Tab"). Even though I like tab() better, not as many packages to import this way

Comment (ccaprio97) [reply query-tool/src/Piipan.QueryTool/package.json:17]: I can do that but some of the tests are using this package. should i just change all the tests to use the realPress thing?

Comment (dborgen2) [reply query-tool/src/Piipan.QueryTool/package.json:17]: Yea, just to be consistent and minimalist here, I think so

**Review by dborgen2 — APPROVED** [review]
👍

File: components/src/Piipan.Components/Forms/TemporaryRadioGroup.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace Piipan.Components.Forms
+{
+    /*
+     * This is a temporary workaround for a .NET bug. This class can be removed after updating to .NET 7 where the bug will be fixed.
+     * https://github.com/dotnet/aspnetcore/pull/40148
+     */
+    public class TemporaryRadioGroup<TValue> : InputRadioGroup<TValue>
+    {
+        private string? _name;
```
Comment (bgfoshay) [inline components/src/Piipan.Components/Forms/TemporaryRadioGroup.cs:11]: Thought: can you add a comment or two documenting why this is necessary and what the bug is you're working around. If you have a link to the .Net 7 fix, that would be great as well

Comment (ccaprio97) [reply components/src/Piipan.Components/Forms/TemporaryRadioGroup.cs:13]: added dan's comment

**Pull request merged**